dotnet/src/Connectors/Connectors.HuggingFace/Core/IEndpointProvider.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+
+namespace Microsoft.SemanticKernel.Connectors.HuggingFace.Core;
+
+internal interface IEndpointProvider
```

Comment:
Do we expect to have multiple implementations of this interface?

Given this is all internal maybe we can simplify things.